app/models/claim.rb

```diff
@@ -74,8 +90,17 @@ def data_for_calculation
       work_items: work_items_for_calculation,
       disbursements: disbursements_for_calculation,
       letters_and_calls: letters_and_calls_for_calculation,
+      youth_court: data['youth_court'] == 'yes',
+      # TODO: CRM457-2306: Amend these as the fields will exist
+      claimed_youth_court_fee_included: data['include_youth_court_fee'],
```

Comment:
Strictly, unless the adjustment flow goes waaaay off piste, this should be `data.fetch('include_youth_court_fee_original', data['include_youth_court_fee'])` - i.e. if there is an "_original" value use that, otherwise use the default value